assets/dev/js/editor/utils/conditions.js

```diff
@@ -34,6 +34,36 @@ Conditions = function() {
 		}
 	};
 
+	/**
```

Comment:
Move to something like `ControlConditions`

Reply:
This method specifically could stay in the regular `conditions file` since it doesn't have any explicit use of control models.
I reverted the `check()` method to not use controls, and created a `ControlConditions` class with a controls-oriented `check()` method.

Reply:
I also converted the Conditions class from an ES5 function to an ES6 class